app/controllers/vehicles_controller.rb

```diff
@@ -6,7 +6,7 @@ def new
   end
 
   def create
-    @vehicle = Vehicle.create(vehicle_params)
+    @vehicle = CreateVehicle.create(vehicle_params)
```

Comment:
Well… If I were you I liked to put this in the controller itself and I would test the controller.

It is difficult subject to talk about because a lot of people like services for everything you can do, but I don't.